nawab.py

```diff
@@ -22,9 +22,9 @@ def twitter_bot_run(data, dirpath, level):
     tw_bot.nawab_retweet_tweet(api)
 
 
-def tg_bot_run(data, dirpath,level):
+def tg_bot_run(data, dirpath,level, isretweet=False, manual_tweet=False):
```

Comment:
So as discussed in the meeting we only need one argument that is -r for auto retweet. You can change the name of the variable isretweet.